crutest/src/cli.rs

```diff
@@ -1076,14 +1105,14 @@ pub async fn start_cli_server(
                         },
                         Some(cmd) => {
                             process_cli_command(
-                                guest,
-                                &mut fw,
-                                cmd,
-                                &mut ri,
-                                &mut wc_filled,
-                                verify_input.clone(),
-                                verify_output.clone()
-                            ).await?;
+                                    guest,
```

Comment:
Is this `rustfmt`'s preference?

Reply:
Looks like no.
I originally changed this section (where is was all shifted right), but then reverted it.
Seems like `cargo fmt` just left it where it was.

I removed the extra space and `cargo fmt` did not change it back.
I'll push an update with this reverted.